test/com/dotcms/filters/interceptor/jwt/JsonWebTokenInterceptorTest.java

```diff
@@ -47,6 +49,12 @@
  * @author jsanca
  */
 public class JsonWebTokenInterceptorTest {
+	
+	@BeforeClass
+	public static void prepare() throws Exception{
+    	//Setting web app environment
+        IntegrationTestInitService.getInstance().init();
```

Comment:
Talked to @jdotcms . This is true unit but is not succeeding. Let's leave it as true unit and @jdotcms will fix it.